Pod/Classes/BugShaker.swift

```diff
@@ -45,7 +51,13 @@ extension UIViewController: MFMailComposeViewControllerDelegate {
   override public func motionEnded(motion: UIEventSubtype, withEvent event: UIEvent?) {
     if motion == .MotionShake {
       let cachedScreenshot = captureScreenshot()
-      
+
+      if let delegate = BugShaker.Config.delegate {
```

Comment:
Can we move this up to be the very first expression in the `motionEnded` method? That way we won't bother to capture a screenshot that we won't use.